org.eclipse.winery.repository.ui/src/app/wineryComponentExists/wineryComponentExists.component.html

```diff
@@ -28,7 +25,8 @@
         <winery-namespace-selector
             name="artifactNamespace"
             [(ngModel)]="generateData.namespace"
-            (onChange)="checkImplementationExists();">
+            (onChange)="checkImplementationExists();"
+            pattern="^\S*$">
```

Comment:
Why not inlining the pattern into `winery-namespace-selector` or is this not possible?

Reply:
It is also present in the `winery-namespace-selector`, where it is used to display the error message. The problem with Angular is, that the form of the parent component will not automatically invalidate if the `pattern` is missing at this point. Here it is required to automatically disable the `save` and `OK` etc. buttons.

Reply:
Please add following as comment above winery-namespace-selector:

    <!-- pattern parameter is required to enable form validation -->